src/device/usbd.c

```diff
@@ -44,6 +44,11 @@
   #define CFG_TUD_TASK_QUEUE_SZ   16
 #endif
 
+#ifdef __ARMCC_VERSION
+// Supress "statement is unreachable" warning
+#pragma diag_suppress 111
```

Comment:
The warnings are for the `switch(desc_type)` inside `process_get_descriptor` in which there're several `break;` that is unreachable because are `return` statements in both `if` and `else` block. Line 1016, 1033, 1059, 1072, 1086.

Reply:
thank you I see, we could just remove the `break;` they are mostly are there for case/break pair.